### PR TITLE
Move TTS autoplay settings to instructionRedux

### PR DESCRIPTION
**Pull request by maureensturgeon**
This work is part of an effort to combine `teacherSectionsRedux` and `sectionDataRedux` to reduce confusion and duplicated data between the two. 

In this PR we're moving ttsAutoplayEnabled setting into instructions redux since this value in redux is used to determine whether ttsAutoplay should be applied for the level based on the sections that a user is in. It's also been renamed to `ttsAutoplayEnabledForLevel` to distinguish between this value and the field on a section that's also called `ttsAutoplayEnabled`

## Links
- jira ticket: [LP-2179](https://codedotorg.atlassian.net/browse/LP-2179)

## Testing story
Tested that level pages are getting ttsAutoplayEnabled value same as before for student and teacher

## Follow-up work
Move more data out of sectionDataRedux

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by maureensturgeon — COMMENTED** [review]
Actually, I'm going to rename `ttsAutoplayEnabled` to `ttsAutoplayEnabledForParticipant` to clarify between the section level setting and the level setting for the user, which is what this scenario covers. Alternatively we could name it `ttsAutoplayEnabledForLevel`

Update: `-ForLevel` makes more sense actually

File: apps/src/sites/studio/pages/levels/show.js [hunk 1]
```diff
@@ -5,22 +5,24 @@ import {getStore, registerReducers} from '@cdo/apps/redux';
...
   const script = document.querySelector('script[data-level]');
   const config = JSON.parse(script.dataset.level);
 
+  registerReducers({sectionData, instructions});
   // this is the common js entry point for level pages
   // which is why ttsAutoplay is set here
-  registerReducers({sectionData});
   const ttsAutoplayEnabled = config.tts_autoplay_enabled;
-  getStore().dispatch(setTtsAutoplayEnabled(ttsAutoplayEnabled));
+  getStore().dispatch(setTtsAutoplayEnabledForLevel(ttsAutoplayEnabled));
```
Comment (jamescodeorg) [inline apps/src/sites/studio/pages/levels/show.js:25]: I'm not sure that the `forLevel` makes this clearer for me but don't feel strongly about it either way.

**Review by jamescodeorg — APPROVED** [review]
Nice cleanup!

File: apps/src/sites/studio/pages/levels/show.js (hunk 1 above)
Comment (maureensturgeon) [reply apps/src/sites/studio/pages/levels/show.js:25]: What do you think would make it clearer for you? I want to distinguish between a section-level setting and the setting that is applied when the user visits the level

**Pull request merged**